Test/git-issues/git-issue-750.dfy

```diff
@@ -0,0 +1,24 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+method m() {
+
+  var a: array<nat> := new nat[10];
+  var b: array<int> := new int[10];
+  b := a;
+  a := b;
```

Comment:
Indicate in this file that these two lines are expected to give errors.

Reply:
OK